src/manage_sql.c

```diff
@@ -33544,6 +33570,54 @@ modify_target (const char *target_id, const char *name, const char *hosts,
         }
     }
 
+  if (krb5_credential_id)
+    {
+      if (target_in_use (target))
+        {
+          sql_rollback ();
+          return 15;
+        }
+
+      krb5_credential = 0;
+      if (strcmp (krb5_credential_id, "0"))
+        {
+          gchar *type;
+          if (find_credential_with_permission (krb5_credential_id,
+                                               &krb5_credential,
+                                               "get_credentials"))
+            {
+              sql_rollback ();
+              return -1;
+            }
+
+          if (krb5_credential == 0)
+            {
+              sql_rollback ();
+              return 26;
+            }
+
+          type = credential_type (krb5_credential);
+          if (strcmp (type, "krb5"))
+            {
+              sql_rollback ();
```

Comment:
Same here with the free of `type`.

Reply:
Fixed in [0e49e1e](https://github.com/greenbone/gvmd/pull/2332/commits/0e49e1ec1be168cb7aa179f28150dc0a93541d52)